sp800-63-3/sec8_references.md

```diff
@@ -5,52 +5,62 @@
 
-<a name="eIDAS"></a>[eIDAS] European Union, *REGULATION (EU) No 910/2014 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL*, July 23,  2014, available at: <http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2014.257.01.0073.01.ENG>.
+### 2. Standards
 
-<a name="ISO29115"></a>[ISO 29115] International Standards Organization, ISO/IEC 29115 *Information technology -- Security techniques -- Entity authentication assurance framework*, April 1, 2013, available at: <http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=45138>.
+<a name="bcp195"></a>[BCP 195] Sheffer, Y., Holz, R., and P. Saint-Andre, *Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS)*, BCP 195, RFC 7525,DOI 10.17487/RFC7525, May 2015, <https://doi.org/10.17487/RFC7525>.
 
-<a name="ISO29003"></a>[ISO 29003] International Standards Organization, ISO/IEC DIS 29003 *Information technology -- Security techniques -- Identity proofing* (under development)
+<a name="Canada"></a>[Canada] Government of Canada, *Standard on Identity and Credential Assurance*, February 1, 2013, available at: <https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=26776>.
 
-<a name="ISO9241"></a>[ISO 9241-11] International Standards Organization, ISO/IEC 9241-11 *Ergonomic requirements for office work with visual display terminals (VDTs) -- Part 11: Guidance on usability*, 1998, available at: <https://www.iso.org/standard/16883.html>. 
+<a name="ISO9241"></a>[ISO 9241-11] International Standards Organization, ISO/IEC 9241-11 *Ergonomic requirements for office work with visual display terminals (VDTs) — Part 11: Guidance on usability*, March 1998, available at: <https://www.iso.org/standard/16883.html>.
 
-<a name="Canada"></a>[Canada] Government of Canada, *Standard on Identity and Credential Assurance*, February 1, 2013, available at: <https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=26776>.
+<a name="ISO29003"></a>[ISO 29003] International Standards Organization, ISO/IEC DTS 29003 *Information technology — Security techniques — Identity proofing*.
 
-<a name="steiner"></a>[Steiner] Steiner, Peter. "On the Internet, nobody knows you're a dog", *The New Yorker*, July 5, 1993.
+<a name="ISO29115"></a>[ISO 29115] International Standards Organization, ISO/IEC 29115 *Information technology — Security techniques — Entity authentication assurance framework*, April 1, 2013, available at: <http://www.iso.org/iso/home/store/catalogue_tc/catalogue_detail.htm?csnumber=45138>.
 
-<a name="NIST-RMF"></a>[NIST RMF] Risk Management Framework Overview, available at <http://csrc.nist.gov/groups/SMA/fisma/framework.html>.
+<a name="OpenIDConnectCore"></a>[OpenID Connect Core 1.0] Sakimura, N., Bradley, B., Jones, M., de Medeiros, B., and C. Mortimore, *OpenID Connect Core 1.0 incorporating errata set 1*, November, 2014,  Available at: <https://openid.net/specs/openid-connect-core-1_0.html>.
```

Comment:
Per Jim Foti, where it is a DOI link we should not have "available at." He seemed not to have a problem with using it where we did not have a DOI link. Not too heavy a lift I don't think to remove all of them. Up to you...